migrations/000003_products.down.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE IF EXISTS "categories_translations" DROP COLUMN "value";
+ALTER TABLE IF EXISTS "products_translations" DROP COLUMN "value";
+ALTER TABLE IF EXISTS "products_translations"
+  RENAME TO "product_translations";
+-- TODO: add constraints
```

Comment:
I guess in the real world, you wouldn't want to merge a completed up migration without a completed down one.

Reply:
Haha yeah, this is something I need to fix before finishing this. But I do YOLO-development here 😬

Reply:
TODO added, resolved for the moment